### DISCUSSION TITLE
React.js, 스프링 부트, AWS로 배우는 웹 개발 101[6단원 질문]

### DISCUSSION THREAD

**Discussion by rightear01** [fsoftwareengineer/todo-application#153] [질문과 답변]
안녕하세요. 해당 교제로 풀스텍을 조금이나마 공부하고 있는 학생입니다.

기존 리엑트와 스프링부트 관련해서는 gradle의 의존성 버전들을 맞춰가면서 다 진행을 하였는데, AWS 부분은 cli로 일레스틱 빈스톡을 이용한 배포를 못하겠습니다..

계속 찾아보았지만, 시원하게 긁어주는 부분이 없어 시간만 계속 허비하고 있네요...

사용자를 만들고 cli로 eb create --database --elb-tpye application --instance-type t2.micro를 사용하여 일래스틱 빈스톡이 로드 밸런서, 오토 스케이링 그룹, RDS, EC2 등을 구축해 주고 연결해주신다고 하셨었는데, 

10. 1일 부로 변경된 정책이 문제인지는 모르겠으나,
Creating Auto Scaling launch configuration failed Reason: Resource handler returned message: "The Launch Configuration creation operation is not available in your account. Use launch templates to create configuration templates for your Auto Scaling groups.

해당 오류가 발생하여  Launch Configuration을 사용하는 것이 아닌,  Launch templates를 따로 만들어서 cli에서 연결을 하라고 하는데 도무지 잘 모르겠습니다..

해당 메일을 보시고 도와주시면 감사하겠습니다...

**Comment by vondermanzen** [comment]
안녕하세요,

eb create 명령어에 --enable-spot 옵션을 추가해 보세요:

`eb create --database --elb-type application --instance-type t2.micro --enable-spot`

이 옵션으로 Launch Template을 자동으로 생성해 문제를 해결할 수 있습니다.

도움이 되길 바랍니다!
